README

```diff
@@ -188,6 +188,16 @@ Compiler Notes
   for more details:
   https://github.com/open-mpi/ompi/issues/3612
 
+- Compiling Fortran programs using the mpi_f08 module on PowerPC with
+  the PGI (tested 17.5) or XL (tested v15.1.5) Fortran compilers and GNU
+  linker after 2.25.1 and before 2.28 will likely experience runtime failures.
+  This was noticed on Ubuntu 16.04 which uses the 2.26.1 version of ld by
+  default. However, this issue impacts any OS running the impacted
+  version of ld. This GNU linker regression will be fixed in version 2.28.
+  Below is a link to the GNU bug on this issue:
+    https://sourceware.org/bugzilla/show_bug.cgi?id=21306
+  The XL compiler will have a fix for this issue in their next release.
```

Comment:
* s/fix/workaround/ because you just said the bug is in the GNU ld, so XL isn't *fixing* the bug -- it's *working around* the bug
* s/their/its/ because you're not anthropomorphizing the XL compiler 😄 

Can you cite a specific version number of the next XL release?  "Next" is an relative (i.e., amorphous) word to use in a README file that will continue to live on the web for years.